.travis.yml

```diff
@@ -6,14 +6,7 @@ node_js:
   - "8"
   - "6"
 env:
-  - KARMA_VERSION=1.3
-  - KARMA_VERSION=1.4
-  - KARMA_VERSION=1.5
-  - KARMA_VERSION=1.6
-  - KARMA_VERSION=1.7
-  - KARMA_VERSION=2.0
-  - KARMA_VERSION=3.0
-  - KARMA_VERSION=3.1
```

Comment:
Would it be too difficult to do a hybrid solution that remains backwards-compatible?

Reply:
No, that would be possible, i've already thought about that point. I will add it asap.

Btw: The travis env property for the current version of karma must be "KARMA_VERSION=5.2", right ?
If so, I would also add "KARMA_VERSION=4.4".